source/api/processing_api.rst

```diff
@@ -462,6 +462,59 @@ Request body example
       ]
   }
 
+Processing cost
+^^^^^^^^^^^^^^^^
+
+``POST https://api.mapflow.ai/rest/processing/cost``
+
+If you want to find out the cost of processing **without creating or running it**, you can use this method.
+
+Request body example:
+
+.. code:: json
+
+  {
+    "wdId": "8cb13006-a299-4df6-b47d-91bd63de947f", 
```

Comment:
"wdName" is more consistent and easy to read (make sure that it's supported properly)